tests/rules/test_underscore_output.py

```diff
@@ -0,0 +1,132 @@
+"""Test src/bitwarden_workflow_linter/rules/underscore_outputs.py."""
+
+import pytest
+
+from ruamel.yaml import YAML
+
+from src.bitwarden_workflow_linter.load import WorkflowBuilder
+from src.bitwarden_workflow_linter.rules.underscore_outputs import RuleUnderscoreOutputs
+
+yaml = YAML()
+
+
+@pytest.fixture(name="correct_workflow")
+def fixture_correct_workflow():
+    workflow = """\
+on:
+  workflow_dispatch:
+    outputs:
+      registry:
+        value: 'Test Value'
+      some_registry:
+        value: 'Test Value'
+  workflow_call:
+    outputs:
+      registry:
+        value: 'Test Value'
+      some_registry:
+        value: 'Test Value'
```

Comment:
🔨 I'd like to see some tests with a workflow that is only triggered by a `push` and does not have any outputs. I want to make sure that the rule logic works for the cases where there are no outputs. We might also want to test a step that has a run but doesn't have an output as well.

Example: 
```
on:
  push:   
  
jobs:
 ....
```

Reply:
@Eeebru Can you create a separate workflow for the `push` trigger than does not include either the `workflow_dispatch` or `workflow_call`?